src/app/authentication/models/signInModel.go

```diff
@@ -0,0 +1,12 @@
+package models
+
+
+type SignInRequest struct {
+	Email    string `gorm:"column:email" json:"email" validate:"required,email" example:"john.doe@gmail.com"`
+	Password string `gorm:"column:password" json:"password" validate:"required,min=8" example:"password"`
+}
```

Comment:
don't use gorm in same model
create model "user" in src/models/dbModels.go for user table.